x/lockup/keeper/lock_test.go

```diff
@@ -242,10 +242,12 @@ func (suite *KeeperTestSuite) TestLocksLongerThanDurationDenom() {
 }
 
 func (suite *KeeperTestSuite) TestLockTokensAlot() {
+	suite.SetupTest()
+
 	addr1 := sdk.AccAddress([]byte("addr1---------------"))
 	coins := sdk.Coins{sdk.NewInt64Coin("stake", 10)}
 	startAveragingAt := 1000
-	totalNumLocks := 2000
+	totalNumLocks := 5000
```

Comment:
This now works just fine with 20000, that'd take 7 seconds on my laptop. Just kept it lower to not take more time in CI.